src/components/Input/Input.stories.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 import Input from '../../components/Input/Input';
+import WalletInputComponent from '../../components/Input/WalletInput';
```

Comment:
can you fix both of these imports please I dont know why it went two level up and back down
`../../components/Input/Input` === `./Input` right?